### PR TITLE
Trainings list

### PR DESCRIPTION
**Pull request by IPoyda**
http://chewbacca.myjetbrains.com/youtrack/issue/App-20 

training list created with styles

### CODE REVIEW

File: frontend/app/index.html [hunk 1]
```diff
@@ -61,6 +59,10 @@
         <script src="scripts/services/userlistService.js"></script>
         <script src="scripts/directives/pageSelectDirective.js"></script>
         <script src="scripts/directives/entryNumberChangeDirective.js"></script>
+        <script src="scripts/controllers/traininglistCtrl.js"></script>
+        <script src="scripts/services/trainingListService.js"></script>
+        <script src="scripts/controllers/traininglistCtrl.js"></script>
```
Comment (pashapovalov) [inline frontend/app/index.html]: Duplicate strings. Pllease use camelCase anation. I mean trainingListCtrl.

File: frontend/app/scripts/services/pagination/paginationService.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+angular.module('frontendApp').directive('pageSelect',
```
Comment (pashapovalov) [inline frontend/app/scripts/services/pagination/paginationService.js]: Please del this file

File: frontend/app/views/main.html [hunk 3]
```diff
@@ -16,6 +16,42 @@
   <h4>Angular</h4>
   <p>
     AngularJS is a toolset for building the framework most suited to your application development.
+    erlkgmekrmge
```
Comment (pashapovalov) [inline frontend/app/views/main.html]: WTF?! please remove this :)

File: src/main/java/com/exadel/controller/TrainingPageController.java [hunk 4]
```diff
@@ -16,6 +16,8 @@
 @RequestMapping("/trainings/training")
```
Comment (pashapovalov) [inline src/main/java/com/exadel/controller/TrainingPageController.java:16]: Do you really need update this file?

File: src/main/java/com/exadel/controller/TrainingsController.java [hunk 5]
```diff
@@ -16,6 +16,7 @@
 public class TrainingsController {
     public static List<Training> trainings = new ArrayList<Training>();
     private static Logger logger = Logger.getLogger(TrainingsController.class.getName());
+
```
Comment (pashapovalov) [inline src/main/java/com/exadel/controller/TrainingsController.java]: Revert this file please

**Comment by kmarmaliykov** [comment]
do we REALLY need 7 commits here?

**Pull request merged**